src/overlays/actors/ovl_En_Stop_heishi/z_en_stop_heishi.h

```diff
@@ -2,16 +2,46 @@
 #define Z_EN_STOP_HEISHI_H
 
 #include "global.h"
+#include "objects/object_sdn/object_sdn.h"
 
 struct EnStopheishi;
 
 typedef void (*EnStopheishiActionFunc)(struct EnStopheishi*, PlayState*);
 
+#define ENSTOPHEISHI_GET_SWITCH_FLAG(this) (((this)->actor.params) & 0x3F)
+#define ENSTOPHEISHI_GET_C(this) ((((this)->actor.params) >> 0xC) & 0xF)
+
 typedef struct EnStopheishi {
-    /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x110];
-    /* 0x254 */ EnStopheishiActionFunc actionFunc;
-    /* 0x258 */ char unk_258[0x90];
+    /* 0x0000 */ Actor actor;
+    /* 0x0144 */ SkelAnime skelAnime;
+    /* 0x0188 */ Vec3s limbDrawTable[SOLDIER_LIMB_MAX];
+    /* 0x01EE */ Vec3s transitionDrawTable[SOLDIER_LIMB_MAX];
+    /* 0x0254 */ EnStopheishiActionFunc actionFunc;
+    /* 0x0258 */ s16 headZRot;
+    /* 0x025A */ s16 headXRot;
```

Comment:
```suggestion
    /* 0x0258 */ s16 headRotZ;
    /* 0x025A */ s16 headRotX;
```
This style seems more common